test/integration/f22/Makefile

```diff
@@ -36,4 +36,4 @@ remote_sync: remote_setup
 	ssh $(SSH_USER)@$(SSH_HOST) "cd kpatch-test/kpatch && make"
 
 remote_slow: remote_sync
-	ssh $(SSH_USER)@$(SSH_HOST) "cd kpatch-test/kpatch/test/integration/f22 && make slow"
+	ssh $(SSH_USER)@$(SSH_HOST) "cd kpatch-test/kpatch/test/integration/f22 && make local"
```

Comment:
But um, yeah I see your point.  The target is "remote_slow" so why not "make slow"?  You're probably right :-)